.prow.yaml

```diff
@@ -0,0 +1,14 @@
+presubmits:
+  GoogleCloudPlatform/blueprints:
```

Comment:
No need for a mapping of repo -> []presubmit. You can use a []presubmit directly here because the repo is infered.

Reply:
done